src/sentry/tasks/derive_code_mappings.py

```diff
@@ -95,3 +133,61 @@ def get_stacktrace(data: NodeData) -> List[Mapping[str, Any]]:
         return [stacktrace]
 
     return []
+
+
+def get_installation(organization: Organization) -> Tuple[Integration, OrganizationIntegration]:
+    integration = None
+    try:
+        integration = Integration.objects.filter(
+            organizations=organization.id,
+            provider="github",
+        )
+    except Integration.DoesNotExist:
+        logger.exception(f"Github integration not found for {organization.id}")
+        return None, None
+
+    if not integration.exists():
+        return None, None
+
+    integration = integration.first()
+    organization_integration = OrganizationIntegration.objects.filter(
+        organization=organization, integration=integration
+    )
+    if not organization_integration.exists():
+        return None, None
+
+    organization_integration = organization_integration.first()
+    return integration.get_installation(organization.id), organization_integration
+
+
+def set_project_codemappings(
+    code_mappings: List[CodeMapping],
+    organization: Organization,
+    organization_integration: OrganizationIntegration,
+    project: Project,
+) -> None:
+    """
+    Given a list of code mappings, create a new repository project path
+    config for each mapping.
+    """
+    for code_mapping in code_mappings:
+        serializer = RepositoryProjectPathConfigSerializer(
+            data={
+                "project_id": project.id,
+                "stack_root": code_mapping.stacktrace_root,
+                "source_root": code_mapping.source_path,
+                "repo_id": code_mapping.repo,
```

Comment:
I'm not entirely sure this will work since one of them is the ID while the other one is the repo's full name.